internal/consensus/reactor.go

```diff
@@ -640,87 +647,99 @@ OUTER_LOOP:
 
 // pickSendVote picks a vote and sends it to the peer. It will return true if
 // there is a vote to send and false otherwise.
-func (r *Reactor) pickSendVote(ctx context.Context, ps *PeerState, votes types.VoteSetReader) bool {
+func (r *Reactor) pickSendVote(ctx context.Context, ps *PeerState, votes types.VoteSetReader) (bool, error) {
 	if vote, ok := ps.PickVoteToSend(votes); ok {
```

Comment:
Not necessary, but it might be nice to invert the case , since we're in here anyway:

```go
vote, ok := ps.PickVoteToSend(votes)
if !ok {
   return false, nil
}
// ... the rest
```

inspired by https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
fair.